tasks/deploy.rb

```diff
@@ -0,0 +1,37 @@
+#!/opt/puppetlabs/puppet/bin/ruby
+require 'json'
+
+params       = JSON.parse(STDIN.read)
+
+# Munge items into arrays of things to deploy
+environments = params['environments'] || params['environment']
+environments = environments.split(',') if environments
+
+modules      = params['modules'] || params['module']
+modules      = modules.split(',') if modules
+
+messages   = []
+exitstatus = 0
+if environments
+  environments.each do |env|
+    messages << "Deploying environment #{env}."
+    `/opt/puppetlabs/bin/r10k deploy environment #{env} --puppetfile`
```

Comment:
It might be worthwhile to allow an optional path parameter, for those who do things their own way.